_scripts/webpack.web.config.js

```diff
@@ -160,7 +163,17 @@ if (isDevMode) {
     })
   )
 } else {
+  const processLocalesPlugin = new ProcessLocalesPlugin({
+    compress: false,
+    inputDir: path.join(__dirname, '../static/locales'),
+    outputDir: 'web/static/locales',
```

Comment:
I believe this will generate an output at `./dist/web/web/static/locales` because `web` is already defined as part of the `output` path above.
```javascript
// line 22 of _scripts\webpack.web.config.js
  output: {
    publicPath: '',
    path: path.join(__dirname, '../dist/web'),
```
![image](https://user-images.githubusercontent.com/106682128/191867193-41ea09c1-4ca8-4524-908d-a89139500352.png)

Reply:
Sorry about that, fixed it now.